posthog/hogql_queries/web_analytics/external_clicks.py

```diff
@@ -102,15 +101,9 @@ def _order_by(self, columns: list[str]) -> list[ast.OrderExpr] | None:
         return [
             expr
             for expr in [
-                ast.OrderExpr(expr=ast.Field(chain=[column]), order=direction)
-                if column is not None and column in columns
-                else None,
-                ast.OrderExpr(expr=ast.Field(chain=["context.columns.visitors"]), order=direction)
-                if column == "context.columns.visitors"
-                else None,
-                ast.OrderExpr(expr=ast.Field(chain=["context.columns.clicks"]), order=direction)
-                if column == "context.columns.clicks"
-                else None,
+                # Primary sorting column. We always have a default sort
+                ast.OrderExpr(expr=ast.Field(chain=[column]), order=direction) if column in columns else None,
+                # Always add URL as final sort
                 ast.OrderExpr(expr=ast.Field(chain=["context.columns.url"]), order="ASC"),
```

Comment:
@rafaeelaudibert I think that now that we have a default column, those middle ones are no longer required. 

Maybe that was like that because it was similar to the `web_stats_table` which has more possibilities? 🤔 But here we only have the `clicks|visitors` as select options to sort by as aliases, so it's ok like that IMHO.

Reply:
Yeah, I think this is all right. You could've simply reordered this code to put clicks first and then visitors (we had it the other way to keep it similar to the rest of the code, but that wasn't a requirement) but this is good as well.